src/style.css

```diff
@@ -0,0 +1,102 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-big {
+  height: 70px;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
```

Comment:
You can use shorthand property instead
https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax
Please fix all cases